Hackney.Shared.HousingSearch/Domain/Transactions/Sender.cs

```diff
@@ -4,6 +4,10 @@ namespace Hackney.Shared.HousingSearch.Domain.Transactions
 {
     public class Sender
     {
+        public Sender()
```

Comment:
Why do we need empty constuctors here?

Reply:
I think the main idea here is to have read-only domain models and to transfer responsibility of object creation to the `Create` method. This is requirenments from @amidomichael